contracts/withdrawal-lock/src/entry.rs

```diff
@@ -114,8 +114,8 @@ pub fn main() -> Result<(), Error> {
                     let custodian_lock = load_cell_lock(custodian_cell_index, Source::Output)?;
                     let custodian_lock_args = {
                         let args: Bytes = custodian_lock.args().unpack();
```

Comment:
We should check the first 32 bytes here, and return an error if the args[..32] isn't match the rollup_type_hash

Reply:
Yes, this should be checked.